code/game/atoms.dm

```diff
@@ -252,8 +252,10 @@ directive is properly returned.
 /atom/proc/relaymove()
 	return
 
-/atom/proc/ex_act()
-	return
+/atom/proc/ex_act(severity, target)
+	set waitfor = FALSE
```

Comment:
```suggestion
```
This part is not needed for our explosion.

Reply:
If you are adding `CHECK_TICK` further down, this may be necessary.
But I would remove those as well until there's a proper explosion port.